src/components/welcome/MyAccounts/__tests__/useSafeOverviews.test.ts

```diff
@@ -0,0 +1,47 @@
+import useSafeOverviews from '../useSafeOverviews'
+import * as balances from '@/hooks/loadables/useLoadBalances'
+import * as sdk from '@safe-global/safe-gateway-typescript-sdk'
+import * as useWallet from '@/hooks/wallets/useWallet'
+import * as store from '@/store'
+import type { Eip1193Provider } from 'ethers'
+import { renderHook } from '@testing-library/react'
+import { act } from 'react-dom/test-utils'
+
+jest.spyOn(balances, 'useTokenListSetting').mockReturnValue(false)
+jest.spyOn(store, 'useAppSelector').mockReturnValue('USD')
+jest
+  .spyOn(useWallet, 'default')
+  .mockReturnValue({ label: 'MetaMask', chainId: '1', address: '0x1234', provider: null as unknown as Eip1193Provider })
+
+describe('useSafeOverviews', () => {
+  it('should filter out undefined addresses', async () => {
```

Comment:
Nit: whilst I can tell that the code is correct, I'd add a test for `undefined` `chainId` values as well - those are the errors we're seeing in the logs.